Makefile

```diff
@@ -47,6 +48,14 @@ release:
 	twine upload --verbose dist/*whl dist/*zip
 	# pip install podman-py
 
+.PHONY: docs
+docs:
+	mkdir -p build/docs/source
+	cp -R docs/source/* build/docs/source
+	sphinx-apidoc -f -o build/docs/source podman podman/tests podman/api_connection.py podman/containers podman/images \
```

Comment:
need to add sphinx to test-requirements or something to ensure it's available. Would be nice to also have a tox docs target but i can add that later

Reply:
@mwhahaha I added it to `requirements.txt` since that is where `setup` and `wheel` were already called out. I'll let you add the tox target.

Reply:
I don't see the update in the latest commit